onetbb build failure (manylinux x86_64, ubuntu-noble)

```
    [onetbb-2022.0.0]   [spkg-install] [ 13%] Linking CXX shared library ../../gnu_13.3_cxx11_64_relwithdebinfo/libtbb.so
    [onetbb-2022.0.0]   [spkg-install] lto1: internal compiler error: in get_token, at opts-common.cc:2123
    [onetbb-2022.0.0]   [spkg-install] Please submit a full bug report, with preprocessed source.
    [onetbb-2022.0.0]   [spkg-install] See <http://bugzilla.redhat.com/bugzilla> for instructions.
    [onetbb-2022.0.0]   [spkg-install] lto-wrapper: fatal error: /opt/rh/gcc-toolset-13/root/usr/bin/g++ returned 1 exit status
    [onetbb-2022.0.0]   [spkg-install] compilation terminated.
    [onetbb-2022.0.0]   [spkg-install] /opt/rh/gcc-toolset-13/root/usr/libexec/gcc/x86_64-redhat-linux/13/ld: error: lto-wrapper failed
    [onetbb-2022.0.0]   [spkg-install] collect2: error: ld returned 1 exit status
    [onetbb-2022.0.0]   [spkg-install] make[5]: *** [src/tbb/CMakeFiles/tbb.dir/build.make:597: gnu_13.3_cxx11_64_relwithdebinfo/libtbb.so.12.14] 
```
https://github.com/passagemath/passagemath/actions/runs/12116558525/job/33822055229#step:11:15059

--- a/build/pkgs/onetbb/spkg-install.in
+++ b/build/pkgs/onetbb/spkg-install.in
@@ -1,6 +1,7 @@
 cd src
 mkdir build
 cd build
-sdh_cmake -DTBB_STRICT=off ..
+# TBB_ENABLE_IPO=OFF is a workaround for https://github.com/passagemath/passagemath/issues/509
+sdh_cmake -DTBB_STRICT=off -DTBB_ENABLE_IPO=OFF ..
 sdh_make
 sdh_make_install

